### PR TITLE
[0.17] Release preparations for 0.17.2

### PR DESCRIPTION
**Pull request by fanquake**
Initial preparations for a 0.17.2 release. Needs a look over by laanwj (especially translations). Also need to check that everyone that should be included in the release notes has been included.

#15445 is the last open [0.17.2 milestone](https://github.com/bitcoin/bitcoin/issues?q=is%3Aopen+is%3Apr+milestone%3A0.17.2) tagged PR. I'm not sure that it's a requirement though.

### CODE REVIEW

**Comment by Sjors** [comment]
If possible, I'd like to include #16639

**Comment by laanwj** [comment]
Concept ACK, haven't checked the translation changes in detail but looks good to me on first glance

File: doc/man/bitcoin-qt.1
Comment (MarcoFalke) [inline doc/man/bitcoin-qt.1]: why is this removed?

Comment (fanquake) [reply doc/man/bitcoin-qt.1]: Thanks, that removal was accidental, as miniupnpc wasn't available when i compiled the 0.17.2 binaries. Fixed up now.

**Comment by GChuf** [comment]
Translations for 0.17 cleaned and updated on transifex.
Be advised, some translations were merged. There were also some translations which were just duplicates and didn't add anything when merging them. The result is less language files, which are in turn more complete.

p.s. I'd still like to further tidy up the translation on transifex, but i'm waiting for laanwj's answer. It's a mess with duplicates everywhere, and translations for locales instead of languages in some cases. Cleaning up further for 0.17 and 0.18 would only make a bigger mess on transifex since I can't move 0.14, 0.15 and 0.16 versions around.

**Comment by fanquake** [comment]
I've updated this to pull the latest Transifex changes for `0.17`.

> Cleaning up further for 0.17 and 0.18 would only make a bigger mess on transifex since I can't move 0.14, 0.15 and 0.16 versions around.

I'd suggest just concentrating on the upcoming 0.19 release. "Cleaning up" anything more for 0.17 and even 0.18 is going to be low impact

**Comment by GChuf** [comment]
> "Cleaning up" anything more for 0.17 and even 0.18 is going to be low impact

I'm well aware of that, I wasn't going to clean the older translations, just move some of them from locales to languages - since I can't do that, moving languages around can make a mess on transifex. Languages which appeared to be complete (100% translated in every branch) would then be only ~80% translated, even though 017x and 018x would be at 100%. I'd rather move all translations together, so that the older translations stay on transifex as examples.

///

I took a quick look at the translations, it seems that the _bitcoin_si.ts_ is wrong - the translations are in another language. I already fixed it on transifex. Please either remove the file or pull the translations from transifex again if it's not too much work.

**Comment by laanwj** [comment]
>  since I can't move 0.14, 0.15 and 0.16 versions around.

Please don't spend time on those versions, their translations will never end up in a release anymore. Only the last two major releases have active translation, and I agree with @fanquake that it only makes sense to focus on the latest release (0.18, translations will open for 0.19 in a few days) for major changes.

**Review by laanwj — APPROVED** [review]
ACK 0bd04c5dd49e1ebc1ffd1db1cc8c42345924865d

**Comment by laanwj** [comment]
> but i'm waiting for laanwj's answer

Sorry, answer to what are you waiting for?

**Comment by GChuf** [comment]
@laanwj sorry if I wasn't clear enough. I'm not spending time on 0.14 - 0.16.
I'd just like to tidy up translations on transifex and move translations from locales to languages ((cs_CZ) to (cs) for example), because we don't need a locale for czech language.
To do this in a clean way, I'd also have to move 0.14-0.16 translations around (or just delete them), which I can't at the moment.

I was asking you for permissions/credentials to do that, but I can't exactly remember when and where.
I've also sent you a message on transifex a while ago, asking for coordinator/reviewer role for slovenian language (si) and (si_SI) (account name _gasper.cefarin_).

I've already done a lot of cleaning and some reorganizing on traqnsifex- you'll see the bump in edited/translated words on transifex dashboard for August.

**Comment by laanwj** [comment]
@gchuf OK, I don't really have control over the transifex organization to deal out credentials AFAIK, apart from making people "maintainer". The person who created it, "seone" seems to have gone MIA, maybe @luke-jr knows more. I do not have access to the "team" pages, for example.

Maybe for the next version it'd make sense to move to a new transifex organization.

**Comment by laanwj** [comment]
I've managed to remove the 0.14-0.16 resources. As for the other questions, see #16754.

**Pull request merged**
